lib/src/core/metadata/directives.dart

```diff
@@ -722,7 +727,7 @@ class DirectiveMetadata extends InjectableMetadata {
 ///
 /// ## Lifecycle hooks
 ///
-/// When the component class implements some [../../guide/lifecycle-hooks.html]
+/// When the component class implements some [lifecycle-hooks](docs/guide/lifecycle-hooks.html)
```

Comment:
They link to [angular.io/dart](https://angular.io/docs/dart). URL adjustment is done when the API docs are generated.